pages/common/svn-changelist.md

```diff
@@ -0,0 +1,24 @@
+# svn changelist
+
+> Associate a changelist with a set of files.
+> More information: <https://subversion.apache.org>.
```

Comment:
Could you please add a more detailed link (if possible)?

Reply:
Sure, can I link to the unofficial but recommended docs at [svnbook](http://svnbook.red-bean.com/en/1.7/svn.advanced.changelists.html)?
See: https://subversion.apache.org/docs/#user-manuals

Reply:
Hmm..."unofficial but recommended"&mdash;@navarroaxel is this OK to do?

Reply:
@navarroaxel, did you see these comments?

Reply:
Yep, I think the new URL is OK.